### PR TITLE
feat: add route delete command

### PR DESCRIPTION
**Pull request by mscb402**

### CODE REVIEW

File: cmd/resource/delete.go [hunk 1]
```diff
@@ -46,6 +46,23 @@ var (
 				output.Errorf("Failed to delete service: %s", err.Error())
 			}
 		},
+		"api": func(id sdk.ID, args ...any) {
```
Comment (tokers) [inline cmd/resource/delete.go:49]: Use "route".

File: cmd/resource/delete.go [hunk 2]
```diff
@@ -46,6 +46,23 @@ var (
...
+			if len(args) == 0 {
+				output.Errorf("Please specify the service id")
+			}
+			serviceID, ok := args[0].(sdk.ID)
+			if !ok {
+				output.Errorf("Please specify the correct service id")
+			}
+
+			if err := cloud.DefaultClient.DeleteAPI(cluster.ID, serviceID, id); err != nil {
+				output.Errorf("Failed to delete api: %s", err.Error())
```
Comment (tokers) [inline cmd/resource/delete.go:63]:
```suggestion
				output.Errorf("Failed to delete route: %s", err.Error())
```

**Review by Hazel6869 — APPROVED** [review]

File: cmd/resource/delete.go (hunk 2 above)
Comment (mscb402) [reply cmd/resource/delete.go]: fixed

File: cmd/resource/delete_test.go [hunk 3]
```diff
@@ -95,3 +96,121 @@ func TestServiceDelete(t *testing.T) {
 		})
 	}
 }
+
+func TestSAPIDelete(t *testing.T) {
```
Comment (tokers) [inline cmd/resource/delete_test.go:100]:
```suggestion
func TestDeleteRoute(t *testing.T) {
```

**Review by tokers — APPROVED** [review]

**Pull request merged**